app/uploadform.js

```diff
@@ -0,0 +1,51 @@
+define(['jquery','app/messages', 'app/config', 'parsley', 'bootstrap-select', 'app/upload'], function ($,messages,config, parsley, select, upload) {
+  console.log(upload);
+  $(function () {
+    $(".selectpicker").selectpicker();
+    window.Parsley.on('field:error', function() {
+        if (this.$element.is("select")) {
+            this.$element.parent().children('.selectpicker').selectpicker('setStyle', 'alert-danger').selectpicker('refresh');
+        }
+    });
+    window.Parsley.on('field:success', function() {
+        if (this.$element.is("select")) {
+            this.$element.parent().children('.selectpicker').selectpicker('setStyle', 'alert-success', 'add').selectpicker('setStyle', 'alert-danger', 'remove').selectpicker('refresh');
+        }
+    });
+    $('select').on('changed.bs.select', function (e) {
+      $(this).selectpicker('setStyle', 'alert-success', 'add').selectpicker('setStyle', 'alert-danger', 'remove').selectpicker('refresh');
+      $(this).parent().children(".parsley-errors-list").html("");
+    });
+    $('select').on('rendered.bs.select', function (e) {
+      $(this).parent().removeClass("parsley-error");
+    });
+
+    // Initiate form error handling
+    $('#uploadForm').parsley().on('field:validated', function() {
+      var ok = $('.parsley-error').length === 0;
```

Comment:
This is the second time I see you using this check to verify if the error is not zero. Consider adding a function.